test/integration/swift/manifest.yaml

```diff
@@ -10,10 +10,6 @@ deployVariables:
     value: "testapp"
   - name: "IMAGENAME"
     value: "host.minikube.internal:5001/testapp"
-  - name: "CPULIMIT"
```

Comment:
These are required to have the swift integration tests run correctly

Reply:
restored, must have been a merge issue